app/tests/repositories/test_answer_repository.py

```diff
@@ -0,0 +1,130 @@
+# @author Lorenzo Varese
+# @created 2023-06-30
+
+import unittest
+import uuid
+from unittest.mock import MagicMock
+
+from app.stuquiz.entities.answer import Answer
+from app.stuquiz.repositories.answer_repository import AnswerRepository
+from app.tests.repositories.test_constants import TEST_ANSWER
```

Comment:
Let's put all the test constants inside the test class, without depending on external file. 
In this way the test class does not depend on a class with random and unlinked constants.